projects/schematics/src/migrations/migrations.json

```diff
@@ -50,6 +50,11 @@
       "version": "4.0.0",
       "factory": "./4_0/removed-public-api-deprecations/removed-public-api-deprecation#migrate",
       "description": "Comment about usage of removed public api"
+    },
+    "migration-v4-rename-symbol-01": {
+      "version": "4.0.0",
+      "factory": "./4_0/rename-symbol/rename-symbol#migrate",
+      "description": "Rename symbols that were moved/changed in majors"
```

Comment:
Since this migration script works only with import symbols, we should be more specific in the description in describe it as such.